### PR TITLE
Fixed conditionals and column name

### PR DESCRIPTION
**Pull request by Boorclark**
Fixes Issue #565 

Changes Made:
 - Changes JS conditionals 
 - Completed --> Passed

### CODE REVIEW

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
I think a better fix is to have 3 options in the select box, because we need to be able to tell a difference between No and never updated. Whether they are choosing yes or no, we still need a date.

File: app/logic/volunteers.py [hunk 1]
```diff
@@ -99,13 +99,16 @@ def setUserBackgroundCheck(user, bgType, checkPassed, dateCompleted):
...
+        createLog(f"Marked {user.firstName} {user.lastName}'s background check for {bgType} as empty.")
     else:
-        createLog(f"Marked {user.firstName} {user.lastName}'s background check for {bgType} as incompleted.")
+        if not dateCompleted:
+            dateCompleted = None
+        update = BackgroundCheck.create(user=user, type=bgType, passBackgroundCheck=int(checkPassed), dateCompleted=dateCompleted)
+        if bool(checkPassed):
+            createLog(f"Marked {user.firstName} {user.lastName}'s background check for {bgType} as completed.")
+        else:
+            createLog(f"Marked {user.firstName} {user.lastName}'s background check for {bgType} as incompleted.")
```
Comment (BrianRamsay) [inline app/logic/volunteers.py]: They are marking them as "Passed" and "Not Passed" now, not complete or incomplete

File: app/static/js/volunteerProfile.js [hunk 2]
```diff
@@ -87,10 +87,11 @@ $(document).ready(function(){
...
     let checkPassed = $("[data-id=" + bgCheckType + "]").val()
 
-    if (checkPassed == 0 && bgDate != '' ) {
-        displayMessage("Y/N<br>Empty!", "danger")
+    if (checkPassed == '' && bgDate != '') {
+        displayMessage("Passed<br>Empty!", "danger")
         return
-    } else if(bgDate == '' && checkPassed != 0) {
+    }
+    if (checkPassed && bgDate == '' ) {
```
Comment (BrianRamsay) [inline app/static/js/volunteerProfile.js]: Does this work with checkPassed = 0?

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/logic/volunteers.py (hunk 1 above)
Comment (bledsoef) [reply app/logic/volunteers.py]: resolved

File: app/static/js/volunteerProfile.js (hunk 2 above)
Comment (bledsoef) [reply app/static/js/volunteerProfile.js]: resolved

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**